components/src/atoms/ListItem/ListItemChildren/ListItemDescriptor.tsx

```diff
@@ -0,0 +1,32 @@
+import * as React from 'react'
+import { Flex } from '../../../primitives'
+import { DIRECTION_ROW, JUSTIFY_SPACE_BETWEEN } from '../../../styles'
+import { SPACING } from '../../../ui-style-constants'
+
+interface ListItemDescriptorProps {
+  type: 'default' | 'mini'
+  description: JSX.Element
+  content: JSX.Element
+}
+
+export const ListItemDescriptor = (
+  props: ListItemDescriptorProps
+): JSX.Element => {
+  const { description, content, type } = props
+  return (
+    <Flex
+      flexDirection={DIRECTION_ROW}
+      gridGap={SPACING.spacing8}
+      width="100%"
+      justifyContent={type === 'mini' ? JUSTIFY_SPACE_BETWEEN : 'none'}
+      padding={
+        type === 'mini'
+          ? `${SPACING.spacing8} ${SPACING.spacing4}`
```

Comment:
https://www.figma.com/design/WbkiUyU8VhtKz0JSuIFA45/Feature%3A-Protocol-Designer-Phase-1?node-id=4759-89854&m=dev

```suggestion
          ? `${SPACING.spacing4} ${SPACING.spacing8}`
```